deploy/evmos/121_deploy_EvmosBTCPool.ts

```diff
@@ -0,0 +1,58 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { execute, get, getOrNull, log, read, save } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // Manually check if the pool is already deployed
+  const SaddleEvmosBTC = await getOrNull("SaddleEvmosBTC")
+  if (SaddleEvmosBTC) {
+    log(`reusing "EvmoswBTCTokens" at ${SaddleEvmosBTC.address}`)
+  } else {
+    // Constructor arguments
+    const TOKEN_ADDRESSES = [
+      (await get("WBTC")).address,
+      (await get("RENBTC")).address,
+    ]
+    const TOKEN_DECIMALS = [8, 8]
```

Comment:
The decimals seems different than what is actually is

https://evm.evmos.org/token/0xF80699Dc594e00aE7bA200c7533a07C1604A106D/token-transfers
https://evm.evmos.org/token/0xb1a8C961385B01C3aA782fba73E151465445D319/token-transfers

Reply:
Its the same as here: https://github.com/saddle-finance/saddle-contract/blob/e4556a3a0ef47d9b09aab699c0b5ab804dcfabb2/deploy/mainnet/100_check_BTCPoolTokens.ts#L8-L9

Reply:
Apparently evmos explorer is lagging. Will add a check for this.